build.gradle

```diff
@@ -111,8 +112,8 @@ subprojects {
     compileOnly 'org.projectlombok:lombok'
   }
 
-  sourceCompatibility = JavaVersion.VERSION_11
-  targetCompatibility = JavaVersion.VERSION_11
+  sourceCompatibility = "1.8"
```

Comment:
no need to change, I just put it here if you want to follow the same way of configuring like java 11, then it is JavaVersion.VERSION_1_8